### ISSUE TITLE
Remove flyout services

### ISSUE THREAD

**Issue by wetneb** [reconciliation-api/specs#42]
Now that the suggest services return descriptions (which was not the case a few years ago), we could consider removing the flyout service which is redundant with these descriptions. Flyout services lead to an awkward user experience where the user must hover a suggest drop-down menu to display the description:
![image](https://user-images.githubusercontent.com/309908/75449668-37e90680-5922-11ea-9d36-a1e7cd292a47.png)

See related issue in Wikidata recon service: https://github.com/wetneb/openrefine-wikibase/issues/77

**Comment by workergnome** [comment]
I think that normalizing the two responses would be really helpful--it's a pain to implement both of them, with their subtle differences.  Not difficult, but annoying.

**Comment by wetneb** [comment]
So you would keep the flyout services in the specs, but unify their output to align them with the descriptions in suggest services?

**Comment by workergnome** [comment]
no--I think resolving them into a single "preview" or "display" service would be the right decision.  The current ones are redundant, and are also not implemented the same way, which means you have to code two different systems to display the same data.

**Comment by wetneb** [comment]
At the moment we have:
* preview services;
* suggest services (which can include descriptions in the suggestions they return);
* flyout services, which return HTML snippets to be displayed next to suggest items.

I propose to remove flyout services altogether - if clients want to display additional previews on results from suggest services, they can use the existing preview service.

**Comment by wetneb** [comment]
Duplicate of #50, sorry!
